src/Npgsql/NpgsqlConnection.cs

```diff
@@ -1732,6 +1742,8 @@ public override DataTable GetSchema()
     /// in the Restrictions collection.
     /// </param>
     /// <returns>The collection specified.</returns>
+    [RequiresUnreferencedCode("Members from serialized types or types used in expressions may be trimmed if not referenced directly.")]
```

Comment:
See the [UnconditionalSuppressMessage] that was already there on parameterless GetSchema above.

Otherwise opened #5293 to maybe enable this.

Reply:
I'll edit the one above too, it has a copypasta message too.